app/javascript/mastodon/features/ui/components/columns_area.js

```diff
@@ -74,9 +88,10 @@ export default class ColumnsArea extends ImmutablePureComponent {
 
   render () {
     const { columns, children, singleColumn } = this.props;
+    const { shouldAnimate } = this.state;
 
     const columnIndex = getIndex(this.context.router.history.location.pathname);
-    const shouldAnimate = Math.abs(this.lastIndex - columnIndex) === 1;
```

Comment:
Shouldn't we animate if we move to the previous / next tab? Might help with discoverability of the feature and - subjective - I think it looks nice.